terra_notebook_utils/drs.py

```diff
@@ -4,11 +4,14 @@
 import json
 import requests
 
-from terra_notebook_utils import WORKSPACE_GOOGLE_PROJECT, WORKSPACE_BUCKET, _GS_SCHEMA
+from terra_notebook_utils import WORKSPACE_GOOGLE_PROJECT, WORKSPACE_BUCKET, TERRA_DEPLOYMENT_ENV, WORKSPACE_NAME, _GS_SCHEMA # noqa
```

Comment:
This can be split into two lines to avoid the `# noqa`:
```
from terra_notebook_utils import (WORKSPACE_GOOGLE_PROJECT, WORKSPACE_BUCKET, TERRA_DEPLOYMENT_ENV, WORKSPACE_NAME,
                                  _GS_SCHEMA)
```